edit/helm.cattle.io.projecthelmchart.vue

```diff
@@ -27,27 +28,35 @@ export default {
 
     this.versionInfo = await this.$store.dispatch('catalog/getVersionInfo', {
       repoType:      'cluster',
-      repoName:      'museum',
-      chartName:     'chartmuseum',
-      versionName: '2.7.0'
+      repoName:      'prometheus-federator',
+      chartName:     'rancher-project-monitoring',
+      versionName: '0.0.1'
```

Comment:
Is there a way to pull this information dynamically? I imagine changes will have to be made when repos and version change.

Reply:
You're right on both fronts in that it'll need to be updated when prometheus-federator hits the rancher chart repo and that there should be some way for versionName to be dynamic... still not sure why it's not loading at all for you though. I'll look into all three of these.

Reply:
oh! I'm a dummy, 'prometheus-federator' is the arbitrary name I gave the repo when I created it which you wouldn't have done. That won't be a problem once Arvind merges his charts into the rancher chart repo. Probably best to hold off on this one until he makes that change and I can adjust my code.